src/data/bucket/symbol_bucket.js

```diff
@@ -205,6 +205,10 @@ class SymbolBucket {
             this.arrays.collisionBox.isEmpty();
     }
 
+    getPaintPropertyStatistics() {
+        return util.extend({}, this.arrays.icon.paintPropertyStatistics, this.arrays.glyph.paintPropertyStatistics);
```

Comment:
Both of these are keyed by layer ID, so doesn't this overwrite icon statistics with glyph statistics?